cli/azd/resources/starter/infra/keyvault.bicep

```diff
@@ -0,0 +1,17 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
```

Comment:
Why is keyvault a required file here? Not all apps need/want it, IMO.

(I've gotten feedback in JS/Python communities that this is actually overly complicated for lots of folks...despite it being perhaps the most secure thing to do)